src/hotspot/share/gc/shenandoah/heuristics/shenandoahAdaptiveHeuristics.cpp

```diff
@@ -56,6 +56,7 @@ const double ShenandoahAdaptiveHeuristics::HIGHEST_EXPECTED_AVAILABLE_AT_END = 0
 const double ShenandoahAdaptiveHeuristics::MINIMUM_CONFIDENCE = 0.319; // 25%
 const double ShenandoahAdaptiveHeuristics::MAXIMUM_CONFIDENCE = 3.291; // 99.9%
 
+// TODO: Provide comment here or remove if not used
```

Comment:
currently used at: https://github.com/openjdk/shenandoah/blob/c0af0d7e04174b2d149c0e5b8c37e3454215ad20/src/hotspot/share/gc/shenandoah/heuristics/shenandoahAdaptiveHeuristics.cpp#L468

Reply:
Some of these may potentially want to be global tunables; may be...